# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gpe-wlancfg/compile.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gpe-wlancfg-0.2.7/main.c.vanilla	2007-08-12 16:35:59.000000000 +0200
+++ gpe-wlancfg-0.2.7/main.c	2007-08-12 16:36:06.000000000 +0200
@@ -25,7 +25,6 @@
 
 #include <gpe/init.h>
 #include <gpe/pixmaps.h>
-#include <gpe/render.h>
 #include <gpe/smallbox.h>
 #include <gpe/errorbox.h>
 #include <gpe/question.h>
